src/DocumentationBrowserViewExtension/NodeDocumentationHtmlGenerator.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Linq;
+using Dynamo.ViewModels;
+
+namespace Dynamo.DocumentationBrowser
+{
+    /// <summary>
+    /// Class responsible for creating the Node Info part of the node documentation. 
+    /// </summary>
+    internal static class NodeDocumentationHtmlGenerator
+    {
+        private const string STYLE_RESOURCE = "Dynamo.DocumentationBrowser.Docs.MarkdownStyling.html";
+
+        /// <summary>
+        /// Creates the Node information section which all nodes have
+        /// even if they dont have additional markdown documentation.
+        /// </summary>
+        /// <param name="e"></param>
+        /// <returns></returns>
+        internal static string FromAnnotationEventArgs(OpenNodeAnnotationEventArgs e)
+        {
+            if (e is null)
+                throw new ArgumentNullException(nameof(e));
+
+            StringBuilder sb = new StringBuilder();
+            sb.AppendLine(DocumentationBrowserUtils.GetContentFromEmbeddedResource(STYLE_RESOURCE));
+            sb.AppendLine(CreateHeader(e));
+            sb.AppendLine(CreateNodeInfo(e));
+
+            return sb.ToString();
+        }
+
+        private static string CreateHeader(OpenNodeAnnotationEventArgs e)
+        {
+            StringBuilder sb = new StringBuilder();
+            sb.AppendLine(string.Format("<h1>{0}</h1>", e.Type));
+            sb.AppendLine(string.Format("<p><i>{0}</i></p>", e.MinimumQualifiedName));
+            sb.AppendLine("<hr>");
+
+            return sb.ToString();
+        }
+
+        private static string CreateNodeInfo(OpenNodeAnnotationEventArgs e)
+        {
+            StringBuilder sb = new StringBuilder();
+            sb.AppendLine("<h2>Node Info</h2>");
```

Comment:
@mjkkirschner Shouldn't these strings be resources?

Reply:
yes - but AFAIK we don't currently have a great way to localize web view content - I guess we could put tokens here in the html, and before displaying them, replace those with data from a resource dictionary in the extension itself?

Reply:
can we file this as an improvement?

Reply:
@mjkkirschner We have localized HTML documents for documentation browser so I guess the user expectation is that this content is localized too. Do you feel it's a big amount of work to externalize the strings here to resources?

Reply:
actually - I recall that @mmisol did some work to enable localization - I think the team may localize the html files directly.

Reply:
hah, thanks for chiming in Martin, can you remind me what the process is for html files currently?

Reply:
On our side, we should make sure HTML files go as embedded resources. That makes it a lot easier for the localization team. However, in this case it is just a string so it should go as a string resource.

Reply:
Thanks!